### PR TITLE
add summary_level argument to pfr_advstats

### LINKED ISSUE #82: `load_pfr_passing()` should load season level adv. passing stats
This functions just loads the combined weekly passing stats instead of the adv. passing stats on season level that is given in the function documentation. 

https://github.com/nflverse/nflreadr/blob/210ed6fc1109225f863508b1c1f277b2fbb0aa88/R/load_nflverse.R#L380-L384

It should load data from here https://www.pro-football-reference.com/years/2021/passing_advanced.htm

The data lives here: https://github.com/nflverse/pfr_scrapR/blob/master/data/pfr_advanced_passing.rds

### PR DESCRIPTION
**Pull request by tanho63**
Resolves #82

### CODE REVIEW

File: R/load_nflverse.R [hunk 1]
```diff
@@ -385,8 +385,9 @@ load_espn_qbr <- function(league = c("nfl", "college"),
 #' @export
 load_pfr_passing <- function(seasons = TRUE){
 
-  load_pfr_advstats(seasons = seasons, stat_type = "pass")
+  cli::cli_warn("`load_pfr_passing()` has been deprecated, please use `load_pfr_advstats(stat_type = 'pass', summary_level = 'season')`")
```
Comment (mrcaseb) [inline R/load_nflverse.R:388]: I am not sure but I think we tried to avoid cli? I think I had it in first and we changed to normal warnings and rlang?

Comment (mrcaseb) [reply R/load_nflverse.R:388]: Would you like to hard deprecate the function? Or maybe use lifecycle?

Comment (mrcaseb) [reply R/load_nflverse.R:388]: Ah nvm. The cli thing was wrong. I don’t know why we don’t use it for the other warnings though.

Comment (tanho63) [reply R/load_nflverse.R:388]: It’s because we brought in CLI to do sitrep, all the other warnings were written the other way first

Comment (mrcaseb) [reply R/load_nflverse.R:388]: Ah ok. So if avoiding lifecycle was on purpose (it would be another dependency) this looks good to me.

**Review by mrcaseb — APPROVED** [review]

**Pull request merged**